static/app/views/alerts/rules/metric/details/body.tsx

```diff
@@ -65,6 +65,39 @@ interface MetricDetailsBodyProps extends RouteComponentProps<{}, {}> {
   selectedIncident?: Incident | null;
 }
 
+export function getPeriodInterval(timePeriod: TimePeriodType, rule: MetricRule) {
```

Comment:
unfortunately we'll need to split these out into another file to  reuse them otherwise we risk dragging in a bunch of stuff